src/templates/events/upcoming-events-template.js

```diff
@@ -28,49 +28,47 @@ const EventsList = ({ title, events }) => {
             path,
             date,
             display_date,
-            // url,
-            // fabricHosted,
           } = event.node.frontmatter;
           return (
-            <Fragment key={`${date}${title}`}>
-              <Subheading>{display_date ? display_date : date}</Subheading>
-              <Heading>
-                <Link to={path}>{title}</Link>
-              </Heading>
-              {/* {!fabricHosted && url && (
-                <Meta
+            <Fragment key={title}>
+              <Subsubheading 
+                style={{
+                  fontWeight: "500",
+                  fontSize: "1rem",
+                  margin: "0 0 0.5rem",
+                }}
+              >
+                {display_date ? display_date : date}
+              </Subsubheading>
+              <Subheading 
+                style={{
```

Comment:
are these styles that belong at a higher level so headings can have a consistent look? these styles don't seem very different that the component's default styles, so perhaps it would be more appropriate to change the base component's style instead of having these one-off, differently styled headings. we want to keep long-term maintenance in mind here.